scripts/claim_withdraw.js

```diff
@@ -9,25 +9,25 @@ const { encodePacked_16_8_128 } = require("../test/snapp_utils.js")
 const MerkleTree = require("merkletreejs")
 const { toHex } = require("../test/utilities.js")
 
-const MongoClient = require("mongodb").MongoClient
-const url = process.env.MONGO_URL ||  "mongodb://localhost:27017/"
-const dbName = process.env.DB_NAME || "dfusion2"
-
-const withdraw_search = async function(db_name, _slot, valid=null, a_id=null, t_id=null) {
-  const db = await MongoClient.connect(url)
-  const collectionName = "withdraws"
-  const dbo = db.db(db_name)
-  const query = { 
-    slot: parseInt(_slot)
-  }
-  if (a_id) query["accountId"] = parseInt(a_id)
-  if (t_id) query["tokenId"] = parseInt(t_id)
-  if (valid) query.valid = true
-
-  console.log(`Querying ${collectionName} with`, query)
-  const res = await dbo.collection(collectionName).find(query).toArray()
-  db.close()
-  return res
+const axios = require("axios")
+const url = process.env.GRAPH_URL || "http://localhost:8000/subgraphs/name/dfusion"
+
+const withdraw_search = async function (_slot, valid = null, a_id = null, t_id = null) {
+  let where_clause = `slot: ${_slot} `
+  if (a_id) where_clause += `accountId: ${parseInt(a_id)} `
+  if (t_id) where_clause += `tokenId: ${parseInt(t_id)} `
+  if (valid) where_clause += `valid: ${valid}`
+
+  const response = await axios.post(url, {
+    query: `query {
+              withdraws(where: { ${where_clause} }) {
+                id, accountId, tokenId, amount, slot, slotIndex, valid
+              }
+            }`
+  })
+  // GraphQL returns "bad" json in that keys are not surrounded by quotes ({"data": {}" vs {data: {}}).
+  // Therefore JSON.parse() fails, since this is only used in tests, using eval should be ok
+  return eval(response.data).data.withdraws
```

Comment:
I am fine with that, but later on, we might use this script for front-ends as well.